### ISSUE TITLE
Chromium 49 crash on Rawhide

### ISSUE THREAD

**Issue by paulcarroty** [lantw44/copr-rpm-spec#1]
https://copr.fedorainfracloud.org/coprs/lantw44/chromium/build/165277/

```
$ chromium-browser 
Received signal 11 SEGV_MAPERR 000000000008
#0 0x55e7081d04ce <unknown>
#1 0x55e7081d08a9 <unknown>
#2 0x7f1e11416410 <unknown>
#3 0x55e708ea5340 <unknown>
#4 0x55e708ea5567 <unknown>
#5 0x55e708ea56c3 <unknown>
#6 0x55e708e97e8c <unknown>
#7 0x55e708e5bb90 <unknown>
#8 0x55e7091d6d2d <unknown>
#9 0x273208806458 <unknown>
  r8: 00000000ffffffff  r9: 0000000000000000 r10: 0000000000000000 r11: 0000000000000246
 r12: 000017c8d7f24310 r13: 000055e70e621004 r14: 000017c8d7f1e020 r15: 0000000000000000
  di: 000017c8d7f1ebc8  si: 000017c8d7db4b00  bp: 000017c8d7dc4d80  bx: 000017c8d7d98090
  dx: 000017c8d7f1ec90  ax: 0000000000000000  cx: 00000cc833300000  sp: 00007ffe511233a0
  ip: 000055e708ea5340 efl: 0000000000010246 cgf: 0000000000000033 erf: 0000000000000006
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000008
[end of stack trace]
Received signal 11 SEGV_MAPERR 000000000008
#0 0x55e7081d04ce <unknown>
#1 0x55e7081d08a9 <unknown>
#2 0x7f1e11416410 <unknown>
#3 0x55e708ea5340 <unknown>
#4 0x55e708ea5567 <unknown>
#5 0x55e708ea56c3 <unknown>
#6 0x55e708e97e8c <unknown>
#7 0x55e708e5bb90 <unknown>
#8 0x55e7091d6d2d <unknown>
#9 0x05ce92406458 <unknown>
  r8: 00000000ffffffff  r9: 0000000000000000 r10: 0000000000000000 r11: 0000000000000246
 r12: 000017c8d7f86310 r13: 000055e70e621004 r14: 000017c8d7f80020 r15: 0000000000000000
  di: 000017c8d7f80bc8  si: 000017c8d7db4b00  bp: 000017c8d7dc4d80  bx: 000017c8d7d98090
  dx: 000017c8d7f80c90  ax: 0000000000000000  cx: 00003a4183300000  sp: 00007ffe511234f0
  ip: 000055e708ea5340 efl: 0000000000010246 cgf: 0074000000000033 erf: 0000000000000006
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000008
[end of stack trace]
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
Received signal 11 SEGV_MAPERR 000000000008
#0 0x55e7081d04ce <unknown>
#1 0x55e7081d08a9 <unknown>
#2 0x7f1e11416410 <unknown>
#3 0x55e708ea5340 <unknown>
#4 0x55e708ea5567 <unknown>
#5 0x55e708ea56c3 <unknown>
#6 0x55e708e97e8c <unknown>
#7 0x55e708e5c376 <unknown>
#8 0x55e708e4eb85 <unknown>
#9 0x55e708d33886 <unknown>
#10 0x25a80220c1fb <unknown>
  r8: 00000000ffffffff  r9: 0000000000000000 r10: 0000000000000000 r11: 0000000000000246
 r12: 000017c8d7f24310 r13: 000055e70e621004 r14: 000017c8d7f1e020 r15: 0000000000000000
  di: 000017c8d7f1ebc8  si: 000017c8d7db4b00  bp: 000017c8d7dc4d80  bx: 000017c8d7d98090
  dx: 000017c8d7f1ec90  ax: 0000000000000000  cx: 00003f9155300000  sp: 00007ffe51122220
  ip: 000055e708ea5340 efl: 0000000000010246 cgf: 0000000000000033 erf: 0000000000000006
 trp: 000000000000000e msk: 0000000000000000 cr2: 0000000000000008
[end of stack trace]

```

**Comment by lantw44** [comment]
Yes, I can reproduce this issue, but I am not going to spend time debugging and reporting it because it doesn't happen on the latest stable version of Fedora, which is Fedora 23 currently. If you have a patch  that can make debugging easier, or you reported this issue and got a fix, I can add it and do a rebuild.

**Comment by paulcarroty** [comment]
This is build problem with GCC 6. Builds from RF works with Clang - http://koji.russianfedora.pro/koji/buildinfo?buildID=2754. Please rebuild.

**Comment by lantw44** [comment]
I started a build for Fedora 24 and rawhide:
https://copr.fedorainfracloud.org/coprs/lantw44/chromium/build/166530/

Fedora 24 build failed because of unsigned packages. I may retry the build after a few days.

**Comment by paulcarroty** [comment]
This is GCC 6 issue: https://github.com/RussianFedora/chromium/issues/4#event-581353279

**Comment by lantw44** [comment]
Resolved by commit e484cae - chromium: Workaround GCC 6 crashes by compiling with clang on Fedora 24 or later.
